### DISCUSSION TITLE
Bug fix release

### DISCUSSION THREAD

**Discussion by robinaly** [encode/httpx#2895] [General]
Hello,

First of all, thanks for this great project!

I recently submitted a small bug fix PR and I was wondering what the release policy of this project is. Or, phrased differently, when I could find a release having this fix on pypi?

Kind regards
Robin

**Comment by karpetrosyan** [comment]
Hi Robin!

We do not have a well-defined release policy yet, but we are planning to strictly follow [SEMVER](https://semver.org/).
Actually, we are not releasing very often, but since we are preparing for the 1.0.0 release, I think the next minor release should be soon!

**Reply by robinaly** [reply]
Thanks. Looking forward to it.
